react/components/FilterBar/FilterTag.js

```diff
@@ -315,8 +317,14 @@ class FilterTag extends PureComponent {
           </Menu>
           {!isEmpty && !isMoreOptions && (
             <div
-              className="flex items-center c-link hover-c-link pointer"
+              className={classNames('flex items-center', {
+                'c-link hover-c-link pointer': !disabled,
+                'bg-disabled': disabled,
+              })}
               onClick={() => {
+                if (disabled) {
+                  return
```

Comment:
That means that the user clicks but nothing happens right? Any given feedback?

Reply:
This is the standard behavior of a button, but as we are using a div instead of a button, we need to do this.